### PR TITLE
Minor docs update for multiple local storage pools for KVM

### PR DESCRIPTION
**Pull request by andrijapanicsb**
minor details updated, for easier configuration and better search results (multiple local storages AND multiple local storage pools)

----
📚 Documentation preview 📚: https://cloudstack-documentation--359.org.readthedocs.build/en/359/

### CODE REVIEW

**Comment by rohityadavcloud** [comment]
Let me try again


@blueorangutan docbuild

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
There is a new UI feature that makes adding Local pools via UI - much easier...
Need to update docs for this as well... 2 ways now to add local pool..,.

Please don't merge, I will update, evetnually...

**Comment by DaanHoogland** [comment]
> Please don't merge, I will update, evetnually...

marking as draft

**Comment by shwstppr** [comment]
@andrijapanicsb is this needed with #366?

**Comment by andrijapanicsb** [comment]
Yes, needed.

**Comment by shwstppr** [comment]
@weizhouapache @DaanHoogland can you please review?

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by shwstppr** [comment]
@blueorangutan docbuild

File: source/adminguide/storage.rst [hunk 1]
```diff
@@ -171,9 +171,12 @@ In order to use multiple local storages, you need to
 
        local.storage.uuid=a43943c1-1759-4073-9db1-bc0ea19203aa,f5b1220b-4446-42dc-a872-cffd281f9f8c
        local.storage.path=/var/lib/libvirt/images,/var/lib/libvirt/images2
+#
 
 #. Restart cloudstack-agent service
 
+    - Storage pools will be automatically created in libvirt by the CloudStack agent
+
 Adding Local Storage Pool via UI
```
Comment (DaanHoogland) [inline source/adminguide/storage.rst:180]:
```suggestion
Adding a Local Storage Pool via UI
```

**Review by shwstppr — APPROVED** [review]
